### DISCUSSION TITLE
Could not render amountNoCurrency in html page, but no error on logs

### DISCUSSION THREAD

**Discussion by naskapal** [firefly-iii/firefly-iii#4386] [General]
**Bug description**
Firefly III version 546, got error message in html, but cannot see the error on logs

**Steps to reproduce**
flushed the artisan cache and visiting /flush

**Extra info**

- euro disabled
- Managed to solve the issue by restarting the container, posting here so people in the future can solve the issue by themselves.

Debug information generated at 2021-02-13 02:59:55 Asia/Jakarta for Firefly III version **5.4.6**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.4.6 |
| Firefly III API | 1.4.0 |
| PHP | 7.4.14 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | d542a374-36c8-49f7-ac76-3d463f2fd6c9 |
| Using docker? | true |
| Telemetry | true |
| Layout | v1 |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | pgsql |
| DB version | 15 (exp. 15) |
| Login provider | eloquent  |
| Trusted proxies (.env) |  |

| User info | Value |
| --- | --- |
| Session start | 2021-02-01 00:00:00 |
| Session end | 2021-02-28 23:59:59 |
| Session first | 2021-02-12 00:00:00 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.150 Safari/537.36 |

**Bonus points**

- I searched and nobody reported this bug before
- screenshot
![image](https://user-images.githubusercontent.com/1687731/107816044-b8f51400-6da6-11eb-910d-402c37146bfd.png)

**Comment by JC5** [comment]
Probably a cache thing, it may happen. Thanks for letting us know!
